src/styles/block/main.scss

```diff
@@ -0,0 +1,58 @@
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $cellSize * 7 + 10;
```

Comment:
There is a potential issue with the calculation of 'max-width'. The expression '$cellSize * 7 + 10' might not be evaluated correctly if '$cellSize' is not a number. Make sure '$cellSize' is defined as a numeric value.